datadog-checks-base/datadog_checks/stubs/aggregator.py

```diff
@@ -45,6 +45,7 @@ def assert_metric(self, name, value=None, tags=None, count=None, at_least=1,
                 continue
 
             if tags and sorted(tags) != sorted(metric.tags):
+                print(metric.tags)
```

Comment:
Remove this ?

Reply:
good catch I thought I removed it, I'll double check if it's the last version of the branch 😕